packages/components/src/mobile/html-text-input/test/index.perf-test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { measurePerformance } from 'reassure';
+import { fireEvent } from '@testing-library/react-native';
+
+/**
+ * Internal dependencies
+ */
+import { HTMLTextInput } from '..';
+
+// Finds the Content TextInput in our HTMLInputView.
+const findContentTextInput = ( screen ) => {
+	return screen.getByLabelText( 'html-view-content' );
+};
+
+describe( 'HTMLTextInput Performance', () => {
+	it( 'measures performance', async () => {
+		const onChange = jest.fn();
+
+		const scenario = async ( screen ) => {
+			// Simulate user typing text.
+			const htmlTextInput = findContentTextInput( screen );
+			fireEvent( htmlTextInput, 'changeText', 'text' );
+
+			// Check if the onChange is called and the state is updated.
+			expect( onChange ).toHaveBeenCalledTimes( 1 );
+			expect( onChange ).toHaveBeenCalledWith( 'text' );
```

Comment:
Similar to an earlier comment, these assertions seem to test internals that may not be necessary to test. The next assertion targeting the display value seems to be more appropriate and provide similar coverage. WDYT?

Reply:
These assertions initially started as linter requirements. Although they don't affect the outcome of the performance results, I think they could be expanded here to help describe what the test is measuring.

Reply:
> Although they don't affect the outcome of the performance results, I think they could be expanded here to help describe what the test is measuring.

I suppose given we are testing this single unit in isolation, asserting the public API (callbacks and props) functions as expected makes sense. 👍🏻